pkg/aggregator/demultiplexer_serverless.go

```diff
@@ -58,7 +58,7 @@ func InitAndStartServerlessDemultiplexer(keysPerDomain map[string][]utils.APIKey
 	}
 	h, _ := hostname.Get(context.Background())
 	config := pkgconfigsetup.Datadog()
-	config.SetWithoutSource("serializer_compressor_kind", "none")
+	config.Set("serializer_compressor_kind", model.SourceAgentRuntime)
```

Comment:
Looks like the value has gone missing. What does this set `serializer_compressor_kind` to?

Reply:
That was a mistake, caused by trying one fix then switching to another and mis-merging the result. It's fixed now.